### PR TITLE
Define ES index and a mapping for the DMARC aggregate report data structure

### PR DESCRIPTION
**Pull request by jsf9k**

### CODE REVIEW

File: dmarc/s3.py [hunk 1]
```diff
@@ -420,23 +517,48 @@ def process_payload(self, content_type, payload):
...
+                                response = requests.post(full_url,
                                                          auth=awsauth,
                                                          json=jsn,
                                                          headers={'Content-Type': 'application/json'},
                                                          timeout=Parser.__Timeout)
                                 # Raises an exception if we didn't get back a
                                 # 200 code
                                 response.raise_for_status()
-                            except requests.exceptions.RequestException as e:
+                            except requests.exceptions.RequestException:
```
Comment (dav3r) [inline dmarc/s3.py:561]: Is there any way to get some text output from this RequestException?

**Review by dav3r — APPROVED** [review]
Strong work @jsf9k - I just had one small question...

File: dmarc/s3.py (hunk 1 above)
Comment (jsf9k) [reply dmarc/s3.py:561]: [`logging.exception`](https://docs.python.org/3/library/logging.html#logging.exception) does exactly that.

Comment (dav3r) [reply dmarc/s3.py:561]: Got it- I see that now, thanks!

**Pull request merged**